README.md

```diff
@@ -42,12 +41,12 @@ OPTIONS:
    -v      Verbose
 ```
    
-Running without the username/password or bad username/password will retrieve the latest public release rather than the latest Plex Pass release.
+Script will auto determine if you installed plexpass or normal version. It will auth using your server's auth token, no more login/password required.
```

Comment:
```suggestion
The script will auto determine if you plexpass or normal version is installed. It will auth using your server's auth token, no login/password required.
```